BankChain/app/src/main/java/nl/tudelft/ewi/ds/bankchain/activities/RecentTransactionsActivity.java

```diff
@@ -60,7 +65,17 @@ public void retrieveRecentTransactions() {
                             "Created session!",
                             Toast.LENGTH_LONG).show();
 
-                    b.listTransactions().thenAccept(ts -> Tools.runOnMainThread(() -> {
+                    Party p = new BunqParty("hello world", 2002);
+                    Account ac = null;
+                    try {
+                        ac = b.listAccount(p).get().stream().findFirst().orElse(new BunqAccount("error", -1, p));
```

Comment:
this is placeholder code and should be refactored into the manager however that is out of scope